src/fparser/two/tests/utils/test_wordclsbase.py

```diff
@@ -145,20 +145,20 @@ def test_wordclsbase_optional_colons():
     result = WORDClsBase.match(token, Name, text, check_colons=True)
     assert str(result) == "('{0}', None)".format(token)
 
-    # token and :: no spaces match
-    text = "{0}::".format(token)
+    # token and name match
+    text = "{0} {1}".format(token, name)
     result = WORDClsBase.match(token, Name, text, check_colons=True)
```

Comment:
In fact, shouldn't this fail to match since `check_colons` is True? The docstring for the match() method says that `check_colons` sets whether or not colons are required (which in itself is confusing - possibly we should re-name the argument?).

Reply:
The docstring is not correct. `check_colons` merely means that it is valid for there to be optional colons there. I've updated the docstring, improved the documentation and changed, 'check_colons' to 'colons'.